src/MediaManagerBundle/Bridge/Chameleon/BackendModule/MediaManagerBackendModule.php

```diff
@@ -630,7 +629,7 @@ protected function renderDetail()
         try {
             $contentHtml = $this->responseVariableReplacer->replaceVariables($contentHtml);
         } catch (TokenInjectionFailedException $e) {
-            $this->returnGeneralErrorMessageForAjax();
+            $this->returnGeneralErrorMessageForAjax('MediaManagerBackendModule: Couldn\'t replace variables');
```

Comment:
We could also provide the exception as the new interface allows an optional exception.

Reply:
ok